proto/autokitteh/triggers/v1/trigger.proto

```diff
@@ -9,7 +9,7 @@ import "buf/validate/validate.proto";
 message Trigger {
   string trigger_id = 1;
   string name = 2;
-  string connection_id = 3;
+  string connection_id = 3; // could be empty
```

Comment:
Just making sure: you are going to add Buf validation, right? This comment isn't really visible/effective for users.

Reply:
adding buffer validation will require either to define schedule in trigger or define data as a formal struct. Therefore there was no buffer validation added. Instead it was added in `sdktypes.Trigger`